frontend/src/scenes/max/maxQuestionSuggestionsLogic.tsx

```diff
@@ -0,0 +1,164 @@
+import { IconBook, IconGraph, IconHogQL, IconPlug, IconRewindPlay } from '@posthog/icons'
+import { actions, connect, kea, path, reducers, selectors } from 'kea'
+import { urls } from 'scenes/urls'
+
+import { productUrls } from '~/products'
+
+import { maxLogic } from './maxLogic'
+import type { maxQuestionSuggestionsLogicType } from './maxQuestionSuggestionsLogicType'
+
+export const maxQuestionSuggestionsLogic = kea<maxQuestionSuggestionsLogicType>([
+    path(['scenes', 'max', 'questionSuggestionsLogic']),
+    connect(() => ({
+        values: [maxLogic, ['question']],
+        actions: [maxLogic, ['setQuestion', 'focusInput', 'askMax']],
+    })),
+    actions({
+        setActiveGroup: (index: number | null) => ({ index }),
+    }),
+    reducers({
+        activeSuggestionGroupIndex: [
+            null as number | null,
+            {
+                setActiveGroup: (_, { index }) => index,
+            },
+        ],
+    }),
+    selectors({
+        suggestionGroups: [
+            () => [], // Kea selector typing hint
+            (): SuggestionGroup[] => QUESTION_SUGGESTIONS_DATA,
+        ],
+
+        activeSuggestionGroup: [
+            (s) => [s.activeSuggestionGroupIndex, s.suggestionGroups],
+            (index, groups): SuggestionGroup | undefined => {
+                if (index === null) {
+                    return undefined
+                }
+                return groups[index]
+            },
+        ],
+    }),
+])
+
+export interface SuggestionItem {
+    label: string
+    // In case the actual question to Max is different from the label
+    content?: string
+}
+
+export interface SuggestionGroup {
+    label: string
+    icon: JSX.Element
+    suggestions: SuggestionItem[]
+    url?: string
+    tooltip?: string
+}
+
+export const QUESTION_SUGGESTIONS_DATA: SuggestionGroup[] = [
+    {
+        label: 'SQL',
+        icon: <IconHogQL />,
+        suggestions: [
+            {
+                label: 'Generate an SQL query to',
+                content: 'Generate an SQL query to ',
+            },
+        ],
+        url: urls.sqlEditor(),
+        tooltip: 'Max can generate SQL queries using your PostHog data and the data warehouse.',
+    },
+    {
+        label: 'Product Analytics',
+        icon: <IconGraph />,
+        suggestions: [
+            {
+                label: 'Create a funnel of the Pirate Metrics (AARRR)',
+            },
+            {
+                label: 'What are the most popular pages or screens?',
+            },
+            {
+                label: 'What is the retention in the last two weeks?',
+            },
+            {
+                label: 'What are the top referring domains?',
+            },
+            {
+                label: 'Calculate a conversion rate for <events or features>',
```

Comment:
This is slightly difficult to understand, specifically the "or features" part, since there is no "feature" concept within PostHog (yet). Maybe we can just write "Calculate a conversion rate for..." and it prefills the input, like you did.